source/base/tensor_product_polynomials.cc

```diff
@@ -700,6 +714,16 @@ AnisotropicPolynomials<dim>::get_n_tensor_pols(
 }
 
 
+template <int dim>
+std::unique_ptr<ScalarPolynomialsBase<dim>>
+AnisotropicPolynomials<dim>::clone() const
+{
+  Assert(false, ExcNotImplemented());
+
+  return 0; // std_cxx14::make_unique<AnisotropicPolynomials<dim>>(*this);
```

Comment:
same here?

Reply:
yes same here. I'm rerunning a build now to see what the errors looked like